website/versioned_docs/version-25.x/MockFunctionAPI.md

```diff
@@ -221,6 +227,12 @@ mock(); // 43
 
 ### `mockFn.mockReturnValueOnce(value)`
 
+Syntactic sugar function for:
```

Comment:
this isn't syntax, it's just another API. Maybe we should say "Shorthand for:" instead?

Reply:
I am referring to the old documentation for writing, eg: `mockFn.mockResolvedValue(value)`

<img width="771" alt="image" src="https://user-images.githubusercontent.com/24872439/156920989-8f552013-88f5-44c6-bef2-32a8277ec5f5.png">

Reply:
hah. well, it's wrong there as well 😀

Reply:
@skychx wanna refresh this? the "syntactic sugar" thing was cleaned up in #13271